### PR TITLE
Make enter jump to cue point if available

### PR DESCRIPTION
**Pull request by sigvef**

### CODE REVIEW

**Comment by aleksanb** [comment]
Does this fill the same need as forcing the playback to always be within the loop?
If so, would that feature be better?

**Comment by sigvef** [comment]
I think there are valid use-cases for playing outside the loop. One example is if you want a specific loop, but are interested in watching a few seconds of "pre-roll" before entering the loop.

File: nin/frontend/app/scripts/directives/keybinding.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ angular.module('nin').directive('keybinding', function(commands, render, demo) {
     },
     '13': function(e) {
       // 'return'
-      commands.jumpToFrame(0);
+      var cuePoint = {};
+      commands.getCuePoint(cuePoint);
```
Comment (aleksanb) [inline nin/frontend/app/scripts/directives/keybinding.js]: How about using a callback function instead?
It's a common pattern and then we won't have to use the proxy object!

**Comment by sigvef** [comment]
ping

**Comment by peterhgombos** [comment]
smooth.

**Comment by sigvef** [comment]
Praise the lord

**Pull request merged**